File: Python/SivitasAkademik.py
# sivitas akademik adalah manusia

from Human import Human

class SivitasAkademik(Human):
    def __init__(self, nik="-", nama="-", jenis_kelamin='-', asal_universitas="-", email_edu="-", fakultas="-"):
        super().__init__(nik,nama,jenis_kelamin)
        self.__asal_universitas = asal_universitas
        self.__email_edu = email_edu
        self.__fakultas = fakultas
    
    def setAsal_universitas(self, asal_universitas):
        self.__asal_universitas = asal_universitas

    def getAsal_universitas(self):
        return self.__asal_universitas

    def setEmail_edu(self, email_edu):
        self.__email_edu = email_edu

    def getEmail_edu(self):
        return self.__email_edu
	
    def setFakultas(self, fakultas):
        self.__fakultas = fakultas
	
    def getFakultas(self):
        return self.__fakultas

File: cpp/SivitasAkademik.cpp
// sivitas akademik adalah manusia

class SivitasAkademik : public Human
{
	private:
		string asal_universitas;
		string email_edu;
		string fakultas;
	public:
		SivitasAkademik(){
			this->asal_universitas = "-";
			this->email_edu = "-";
			this->fakultas = "-";
		}
		SivitasAkademik(string nik, string nama, string jenis_kelamin, string asal_universitas, string email_edu, string fakultas)
		: Human(nik, nama, jenis_kelamin){
			this->asal_universitas = asal_universitas;
			this->email_edu = email_edu;
			this->fakultas = fakultas;
		}
		void setAsal_universitas(string asal_universitas){
			this->asal_universitas = asal_universitas;
		}
		string getAsal_universitas(){
			return this->asal_universitas;
		}
		void setEmail_edu(string email_edu){
			this->email_edu = email_edu;
		}
		string getEmail_edu(){
			return this->email_edu;
		}
		void setFakultas(string fakultas){
			this->fakultas = fakultas;
		}
		string getFakultas(){
			return this->fakultas;
		}
};